src/components/Aside/_aside.scss

```diff
@@ -19,4 +19,41 @@
     justify-content: center;
     width: 100%;
   }
+
+  &.is-split {
```

Comment:
This split logic seems like it's very dependent on the content and should be part of the sharing styles and not the aside itself.

Reply:
The config panel is also split in this way... smells like a pattern to me.

Reply:
It is, but it collapses totally different. The columns do not stack on the config but they do here.

Reply:
Not a show stopper here